.github/label-actions.yml

```
@@ -438,4 +454,23 @@
     If you are a paying Mend.io customer, please tell your support or customer contact that this issue is important to you.
 
 
+    Thanks, the Renovate team
+
+'auto:reduce-complexity':
+  comment: >
+    Hi there,
+
+
+    This discussion is too complex, and we recommend you simplify it to increase chances of getting help and finding a solution.
```

```suggestion
    This discussion is too complex, and we want you to simplify. This way you are more likely to get help or a solution.
```

I suspect that the verb here is _want_ instead of _recommend_. Also we can chop this sentence into two parts.